### PR TITLE
Item type controller

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

**Comment by michaelachrisco** [comment]
Looking now.

File: app/helpers/fagus/item_types_helper.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module Fagus
```
Comment (michaelachrisco) [inline app/helpers/fagus/item_types_helper.rb]: Please remove.

File: spec/features/user_views_a_list_of_item_types_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    scenario 'the user sees a no item types message' do
+      visit item_types_path
+
+      expect(page).to have_content('No item types found')
+    end
+  end
+
+  context 'when there are item types' do
+    let!(:item_types) do
+      [
```
Comment (michaelachrisco) [inline spec/features/user_views_a_list_of_item_types_spec.rb]: You can use: `create_list(:item_types, 2)` to create this list.

File: spec/features/user_views_an_item_type_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
```
Comment (michaelachrisco) [inline spec/features/user_views_an_item_type_spec.rb:2]: Why separate these by 0 item types, + item types, then have a separate file and spec for one item type?

File: spec/features/user_views_a_list_of_item_types_spec.rb (hunk 2 above)
Comment (josephbridgwaterrowe) [inline spec/features/user_views_a_list_of_item_types_spec.rb]: Nice, thanks!

File: spec/features/user_views_an_item_type_spec.rb (hunk 3 above)
Comment (josephbridgwaterrowe) [inline spec/features/user_views_an_item_type_spec.rb:2]: This is the show test and not the index test, or am I confused? :smile:

Comment (michaelachrisco) [inline spec/features/user_views_an_item_type_spec.rb:2]: No you are correct...although that leaves some other holes in some other apps...Carry on.

File: app/views/fagus/item_types/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<h2>Item Types</h2>
+
+<p>
+  <%= link_to 'New', new_item_type_path %>
+</p>
+
+<table>
+  <thead>
+    <tr>
+      <th>Description</th>
```
Comment (michaelachrisco) [inline app/views/fagus/item_types/index.html.erb:10]: Is the index going to include the UUID?

Comment (michaelachrisco) [inline app/views/fagus/item_types/index.html.erb:10]: If the index does not need a UUID, then you can rebase.

Comment (josephbridgwaterrowe) [inline app/views/fagus/item_types/index.html.erb:10]: Nope, I saw that to be more of a drill down value. :smile:

Comment (michaelachrisco) [inline app/views/fagus/item_types/index.html.erb:10]: Ok. rebase and i will merge.

**Comment by josephbridgwaterrowe** [comment]
Thanks.

**Pull request merged**